app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, original_file, new_file = command.split()
```

Comment:
Validate command you code will fails with an undefined error if the command will be one of:
mv file1.txt
mv file1.txt file2.txt file3.txt

Reply:
Also, remame variables 
mv -> command or input_commnad
new_file -> destination_file_path
original_file -> original_file_path